packages/edit-site/src/components/dataviews/default-views.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// DEFAULT_STATUSES is intentionally sorted. Items do not have spaces in between them.
+// The reason for that is to match the default statuses coming from the endpoint
+// (entity request and useEffect to update the view).
+export const DEFAULT_STATUSES = 'draft,future,pending,private,publish'; // All statuses but 'trash'.
+
+const DEFAULT_PAGE_BASE = {
```

Comment:
I don't think we need a different file for this config. It's specific to pages list, so it could live there and shouldn't be part of the dataviews component.